src/main/public/js/cookies-manager.js

```diff
@@ -0,0 +1,192 @@
+$(document).ready(function(){
```

Comment:
I am not sure about using js cookie manager in typescript... I would prefer it to be typescript cookie manager. Otherwise in my opinion it would be a bit of a mess.. in addition there are existing cookie managers or if you prefer write our own. but in typescript :P

Reply:
The cookie manager was generated by gulp, it must gotten into my commits, removed.